docs/reference/glossary.md

```diff
@@ -93,6 +93,27 @@ develop, ship, and run applications
 - The docker daemon process running on the host which manages images and containers
 
 
+## Docker for Mac
+
+[Docker for Mac](https://docs.docker.com/docker-for-mac/) is an easy-to-install,
+lightweight Docker development environment designed specifically for the Mac. A
+native Mac application, Docker for Mac uses the macOS Hypervisor framework,
+networking, and filesystem. It's the best solution if you want to build, debug,
+test, package, and ship Dockerized applications on a Mac. Docker for Mac
+supersedes [Docker Toolbox](#toolbox) as state-of-the-art Docker for macOS.
```

Comment:
@thaJeztah this is an internal link to the Toolbox topic in the Glossary. I think it's worth including some internal x-refs, and I don't really want to send people from the Docker for Mac (or Windows) definitions out to the Toolbox docs because we are trying to move them away from Toolbox. When you are in the Toolbox definitions, I _do_ think it's worth linking out to d4mac and d4win docs as a way of encouraging to use those apps instead. Does that make sense?